### ISSUE TITLE
Moving to php 7.0+?

### ISSUE THREAD

**Issue by HueyNemud** [benhutchins/docker-mediawiki#22]
Hi, are you planing to move from php:5.6 to php:7.0?
Some of the main mediawiki extensions require at least PHP 7.0 in their latest releases, i.e Maps.

I use older versions of Maps & others for now, but this is not very satisfying.

**Comment by benhutchins** [comment]
I agree, this will be important for newer MediaWiki versions. Right now some of the PHP versions for older MediaWiki images are also not ideal, so I'll need to work out a good solution for this, but it's on my radar to resolve within the next month.
